### DISCUSSION TITLE
Hourly Activities

### DISCUSSION THREAD

**Discussion by npcwoody** [jlai/fitness-dashboard#12] [Ideas]
Have you considered adding Hourly Activities? It is one of the things I try to achieve every day. I try to do at least 250 per hour for ten hours a day.  It's on the phone app, and it was on the FitBit Web Dashboard.

**Comment by jlai** [comment]
Yes, although I'd need to get "intraday" access approved by Fitbit first in order to access hourly data (instead of day granularity)

**Comment by npcwoody** [comment]
Thanks. That would be great if you could. I can always refer to the phone app, but it would be easier here.... You probably have members who are hesitant to join github. I was.....

**Reply by jlai** [reply]
Do you have any suggestions on a site besides Github for discussions? There's reddit but I don't know if everyone is willing to create an account there either.

**Reply by npcwoody** [reply]
I'm not sure. I don't do much discussion on the internet. At 72 years old I am not too much into new things. I use FitBit to help with my health issues.  Maybe if you could put a reassuring explanation somewhere in front of the links.....
